docs/coding-style.md

```diff
@@ -27,7 +27,17 @@ Static fields and properties should be placed before instance ones.
 
 Static methods are preferred to be after instance methods.
 
-Individual members must be separated by empty line, except sequence of constants, fields and abstract members. These should not be separated by empty lines.
+Methods which are called by other methods in the same category should be placed below the callers.
```

Comment:
The only issue I have here is the word category. It's not clear at first reading whether it refers to one of the groups from the list above (e.g. constants is a category) or it's more refined (e.g. private static constants is a category). 
`Methods which are called by other methods in the same category should be placed below the callers.`
Maybe accessibility is a better word here, although that doesn't include static/non-static. Not entirely sure about it.

Reply:
Thanks. I'll open an improvement. https://github.com/SonarSource/sonar-dotnet/pull/6787/files